filemgr/src/main/java/org/apache/oodt/cas/filemgr/catalog/Catalog.java

```diff
@@ -324,7 +324,7 @@ ProductPage pagedQuery(Query query, ProductType type, int pageNum)
      * @throws CatalogException
      *             If any error occurs (e.g., the layer isn't initialized).
      */
-    ValidationLayer getValidationLayer();
+    ValidationLayer getValidationLayer() throws CatalogException;
```

Comment:
Let's not change the API here. Instead, we should be doing `null` checks everywhere.